arclight-common/src/main/java/io/izzel/arclight/common/mixin/core/entity/EntityMixin.java

```diff
@@ -599,6 +595,14 @@ private static boolean isLevelAtLeast(CompoundNBT tag, int level) {
         return passengers;
     }
 
+    @Override
+    public void bridge$getRecursivePassengers(boolean playersOnly, Set<Entity> passengers) {
+        this.getRecursivePassengers(playersOnly, passengers);
+    }
+
+    @Shadow
+    protected abstract void getRecursivePassengers(boolean playersOnly, Set<Entity> passengers);
```

Comment:
shadows should be placed at the beginning of the class, inside the @formatter:off block

Reply:
Yep, my bad I didn't notice the methods in there too. Updated!